### ISSUE TITLE
Style props for changing textBody or title default styles

### ISSUE THREAD

**Issue by pisacode** [CodingByJerez/react-native-alert-notification#7]
It would be nice to have a style prop, I will try to open a pr If you can managed to do it on my fork.

**Comment by fessor10** [comment]
Yes, this will be great. I was just thinking about this just now. Can you open a PR for this? 

I'm expecting we can define the font sizes for the toast text body and not be restricted to the default.

@CodingByJerez Is this something you're considering?

**Comment by fessor10** [comment]
Added this quick PR https://github.com/CodingByJerez/react-native-alert-notification/pull/8#issue-1315840603

**Comment by CodingByJerez** [comment]
I added an option to change text sizes [v0.2.0](https://github.com/CodingByJerez/react-native-alert-notification/releases/tag/v0.2.0).

I think in a month I will review the library in total to add option, (I don't have a lot of time lately, a lot of work :/ )
